src/Composer/Repository/PlatformRepository.php

```diff
@@ -154,7 +154,12 @@ protected function initialize()
                     break;
 
                 case 'intl':
-                    $name = 'ICU';
+                    # Add a seperate version for the CLDR library version
+                    $cldrVersion = \ResourceBundle::create('root', 'ICUDATA-curr', false)->get('Version');
+                    $this->addLibrary('cldr', 'The unicode CLDR project', $cldrVersion);
+
+                    $name = 'icu';
+                    $description = 'The ICU unicode and globalization support library';
```

Comment:
We need to support 5.3 and 5.4 which don't have the class present:

```suggestion
                    // Add a seperate version for the CLDR library version
                    if (class_exists('ResourceBundle')) {
                        $icuBundle = \ResourceBundle::create('root', 'ICUDATA-curr', false);
                        $this->addLibrary('cldr', 'The unicode CLDR project', $icuBundle->get('Version'));
                    }

                    $name = 'icu';
                    $description = 'The ICU unicode and globalization support library';
```

Reply:
Good point, I've misread the docs on that. Pushed a fix to make that conditional.